common/djangoapps/student/views.py

```diff
@@ -314,6 +314,8 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
         CertificateStatuses.notpassing: 'notpassing',
         CertificateStatuses.restricted: 'restricted',
         CertificateStatuses.auditing: 'auditing',
+        CertificateStatuses.audit_passing: 'auditing',
+        CertificateStatuses.audit_notpassing: 'auditing',
```

Comment:
@stroilova This mapping controls the display logic in the student dashboard template. `'auditing'` status will cause the certificate to not be displayed (but grading information will be displayed).